test/function_manager.jl

```diff
@@ -101,4 +104,5 @@
     # finally
     #     rmprocs(workers())
     # end
+    Disconnect(client)
```

Comment:
probably better to us `finally` or somethign?

Reply:
Using `finally` would be better. Alternatively you could use a `do`-block like this:
```julia
Connect(JuliaGcsClient("127.0.0.1:6379")) do client
   ...
end
```

Reply:
added a new method with do-syntax